src/components/views/rooms/NotificationBadge.tsx

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 import React from "react";
 import classNames from "classnames";
-import { formatMinimalBadgeCount } from "../../../utils/FormattingUtils";
```

Comment:
This was the only use of `formatMinimalBadgeCount`, so seems good to remove it as well.